### PR TITLE
allow report generation for scheduled reports

### PR DESCRIPTION
**Pull request by pruan-rht**

### CODE REVIEW

File: features/step_definitions/metering.rb [hunk 1]
```diff
@@ -325,3 +325,70 @@
...
+#         "spec" => {
+#           "reportingStart" => opts[:start_time],
+#           "reportingEnd" => opts[:end_time],
+#           "query" => opts[:query_type],
+#           "gracePeriod" => opts[:grace_period],
+#           "runImmediately" => opts[:run_immediately],
+#           "schedule" => schedule
+#         },
+#
+Then /^I generate a metering report with:$/ do |table|
```
Comment (akostadinov) [inline features/step_definitions/metering.rb]: shouldn't this be `When`?

File: lib/openshift/report.rb [hunk 2]
```diff
@@ -21,7 +21,7 @@ def conditions(user: nil, cached: false, quiet: false)
 
     def running?(user: nil, cached: false, quiet: false)
       # the reason 'Scheduled' is an initial state what we
-      conditions.any? { |c| c.type == 'Running' and c.reason != 'Scheduled' }
+      conditions.any? { |c| c.type == 'Running' }
```
Comment (akostadinov) [inline lib/openshift/report.rb]: shold this check that the status is `true` or something? Conditions can be true or false, no?

Comment (akostadinov) [reply lib/openshift/report.rb]: If this is not the case, then you can merge without me if that's blocking you.

File: features/step_definitions/metering.rb (hunk 1 above)
Comment (pruan-rht) [reply features/step_definitions/metering.rb]: updated.

File: lib/openshift/report.rb (hunk 2 above)
Comment (pruan-rht) [reply lib/openshift/report.rb]: @akostadinov, this is correct, we are just checking the word `Running` is there.

**Pull request merged**